apps/laboratory/tests/shared/pages/ModalPage.ts

```diff
@@ -407,7 +407,7 @@ export class ModalPage {
     }
 
     await expect(this.page.getByText(headerTitle)).not.toBeVisible({
-      timeout: 20_000
+      timeout: 30_000
```

Comment:
This one is weird - for last few weeks we had issues on Farcaster tests including this PR.

I've did some investigation and seemed like after pasting OTP, it's not resolving the getUser in 20 secs. But actually it's resolving eventually, it's just taking more. I've updated the timeout slightly and it helped.

What we do in Farcaster is that we reload the secure site if the user opens Farcaster page and go back, just to clear Farcaster state on secure site, we reload it - introduced here: https://github.com/reown-com/appkit/pull/3527/files But tbh I don't really now why just reloading secure site causes requests to be resolved slower.